### DISCUSSION TITLE
Have not received updates on my GitHub Campus experts Application

### DISCUSSION THREAD

**Discussion by jim4067** [community/community#45570] [GitHub Education] [answered]
I applied to the GitHub campus experts program in early September and have yet to receive details on my application.

If the application was not looked at, can it be reviewed under the February applicants?

**Comment by jge162** [comment, marked as answer]
Hello @jim4067 thank you for your post. The GitHub Campus Experts program is a competitive program, and the selection process can take some time. It's possible that your application is still under review, and that you haven't received an update yet.

You can reach out to [GitHub Campus Experts team by email](https://education.github.com/students/experts/terms#:~:text=To%20provide%20us%20with%20notice,to%20education%40github.com.) to inquire about the status of your application. They might be able to give you more information on the timeline for your application review.

click on link above and you can find email in posts, then email and inquire for update.

**Reply by Experience-rookie** [reply]
I'm reaching out regarding an issue I've encountered with my application for the GitHub Campus Experts program. Despite submitting my application and receiving initial communication from the GitHub team about the review process, I haven't received any updates regarding the status of my application.

As per the initial communication, updates were expected by March 19, 2024. However, it's now April 17, 2024, and I'm yet to receive any further communication.

Please I need some updates regarding my application
